app/services/phone_number_capabilities.rb

```diff
@@ -9,6 +9,15 @@ def self.load_config
 
   attr_reader :phone, :phone_confirmed
 
+  def self.translated_international_codes
+    translated_international_codes_data = {}
+    INTERNATIONAL_CODES.each do |k, value|
+      translated_international_codes_data[k] =
+value.merge('name' => I18n.t("countries.#{k.downcase}"))
```

Comment:
no need to stop the merge, but the formatting on this is super funky ... I will make a PR to update

Reply:
https://github.com/18F/identity-idp/pull/6122